www/src/styles/global.css

```diff
@@ -188,8 +188,8 @@
     @apply break-words [direction:ltr] [unicode-bidi:embed] lg:break-normal;
   }
 
-  .markdown code:not(.markdown pre code) {
-    @apply rounded border border-t3-purple-200 bg-t3-purple-200/40 px-0.5 py-[0.5px] dark:border-t3-purple-400/30 dark:bg-t3-purple-400/20;
+  .markdown p div:has( code) {
+    @apply inline rounded border border-t3-purple-200 bg-t3-purple-200/40 px-0.5 py-[0.5px] dark:border-t3-purple-400/30 dark:bg-t3-purple-400/20;
```

Comment:
I made this selector in hurry, there may be some code tags lost because of it, ones which come to mind are code elements in the heading.

```suggestion
  .markdown div:has( > code) {
    @apply inline rounded border border-t3-purple-200 bg-t3-purple-200/40 px-0.5 py-[0.5px] dark:border-t3-purple-400/30 dark:bg-t3-purple-400/20;
```

This is a different selector, if this is the case, if this doesn't work, I can revert to applying the classes through the function.

Reply:
Yea headings are lost

![image](https://user-images.githubusercontent.com/51714798/212097516-b18a9bf1-f18d-4f11-8743-70223a811f40.png)